fighthealthinsurance/static/js/blog_post.tsx

```
@@ -0,0 +1,197 @@
+                    if (!isNaN(dateObj.getTime())) {
+                      return dateObj.toLocaleDateString('en-US', { year: 'numeric', month: 'long', day: 'numeric' });
+                    }
+                  }
+                  // Fallback: show raw date
+                  return metadata.date;
+                })()}
+              </div>
+            )}
+          </div>
+          
+          <div dangerouslySetInnerHTML={{ __html: content }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)